qa/selenium-ide/web-server/testCases/Vehicles

```diff
@@ -84,3 +84,4 @@
 </tbody></table>
 </body>
 </html>
+<!--covers test TST-18-->
```

Comment:
I also put it at the end because there would be another couple of tickets that this test case covers, and having them stacked at the end looked 'right' to me. So no valid reason really :D